app/services/folio/courses.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Folio
+  class Courses
+    class << self
+      delegate :courses, to: :instance
+    end
+
+    def self.instance
+      @instance ||= new
+    end
+
+    attr_reader :folio_client
+
+    def initialize(folio_client: FolioClient.new)
+      @folio_client = folio_client
+    end
+
+    # Write courses from FOLIO to database
+    def sync_courses!
```

Comment:
Do we also need to delete courses from the database that have been removed from FOLIO?

Reply:
@corylown Chris had the same question above so I answered it there. In short, yes, and I should add that.